src/hostcalls.rs

```diff
@@ -414,8 +414,10 @@ pub fn get_property(path: Vec<&str>) -> Result<Option<Bytes>, Status> {
                 } else {
                     Ok(None)
                 }
-            }
+            },
```

Comment:
Nit: this is unnecessary change and it doesn't match the existing style in the codebase.